app/controllers/hosts_controller.rb

```diff
@@ -918,4 +921,8 @@ def host_attributes_for_templates(host)
        host_attributes.include?(k) && !k.end_with?('_ids')
     end.except(:host_parameters_attributes)
   end
+
+  def csv_columns
+    [:id, :name, :operatingsystem, :environment, :model, :hostgroup, :last_report]
```

Comment:
I'm not sure, might be useful to have it just in case but I don't mind removing either.